python/tvm/tir/schedule/schedule.py

```diff
@@ -43,7 +43,7 @@ class BlockRV(Object):
     """A random variable that refers to a block"""
 
 
-ExprRV = PrimExpr  #  A random variable that evaluates to an integer
+ExprRV = Union[PrimExpr]  #  A random variable that evaluates to an integer
```

Comment:
This is exactly the same as `PrimExpr`?

Reply:
It is a workaround for mypy: https://github.com/python/mypy/issues/7866#issuecomment-549454370

This feature is not supported until python 3.10: https://docs.python.org/3.10/whatsnew/3.10.html#pep-613-typealias

Reply:
Hmm I see. Then we can add this issue's link to the comment.